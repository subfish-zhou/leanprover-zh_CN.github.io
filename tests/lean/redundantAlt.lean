def f (x : Nat) : Nat :=
 match x with
 | 0 => 1
 | Nat.succ (Nat.succ x) => 3
 | 2 => 4
 | Nat.succ x => 5
 | 3 => 6
