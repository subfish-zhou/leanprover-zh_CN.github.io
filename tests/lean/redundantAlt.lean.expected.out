redundantAlt.lean:5:1-5:9: error: redundant alternative
redundantAlt.lean:7:1-7:9: error: redundant alternative
